src/openzeppelin/token/erc20/library.cairo

```diff
@@ -34,291 +29,335 @@ end
 #
 
 @storage_var
-func ERC20_name_() -> (name: felt):
+func name_() -> (name: felt):
 end
```

Comment:
IMO storage should not be imported. Contracts using libraries like this one should only rely on functions (including getters and setters) that are in the corresponding namespace. 

Would clash happen for storage with the same name that are part of the dependencies but that are not explicitly imported?

Reply:
> IMO storage should not be imported. Contracts using libraries like this one should only rely on functions (including getters and setters) that are in the corresponding namespace.

This is probably the safer/better design play. I can agree here.

> Would clash happen for storage with the same name that are part of the dependencies but that are not explicitly imported?

Yes with the caveat that the compiler will throw an AssertionError if the same-name storage vars differ in any way—including return value name, variable name, and number of keys. Otherwise, the same-name storage vars will have the same storage address and seen/treated as the same storage.

Reply:
> > Would clash happen for storage with the same name that are part of the dependencies but that are not explicitly imported?
> 
> Yes with the caveat that the compiler will throw an AssertionError if the same-name storage vars differ in any way—including return value name, variable name, and number of keys. Otherwise, the same-name storage vars will have the same storage address and seen/treated as the same storage.

IMO, this is terrible language design ...

I renamed the storage slots to account for that.